conans/test/utils/tools.py

```diff
@@ -542,7 +527,12 @@ def run(self, command_line, assert_error=False):
             self.out = RedirectedTestOutput()  # Initialize each command
             with redirect_output(self.out):
                 with redirect_input(self.user_inputs):
-                    return self.run_cli(command_line, assert_error=assert_error)
+                    if self.requester_class:
```

Comment:
I think you have removed the possibility to run the ``artifactory_ready`` tests against Artifactory, is this intended?

Reply:
That was kind of working because the `TestRequester` was doing a fallback to `requests` when the URL didn't match `fake***` but I changed the code to be explicit and not mock the http_requester.